Assignment.Tests/PingProcessTests.cs

```diff
@@ -0,0 +1,284 @@
+﻿using IntelliTect.TestTools;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Assignment;
+using System.Xml.Linq;
+
+namespace Assignment.Tests;
+
+
+[TestClass]
+public class PingProcessTests
+{
+    PingProcess Sut { get; set; } = new();
+
+    [TestInitialize]
+    public void TestInitialize()
+    {
+        Sut = new();
+    }
+
+    [TestMethod]
+    public void Start_PingProcess_Success()
+    {
+        Process process = Process.Start("ping", "-c 4 localhost");
+        process.WaitForExit();
+        Assert.AreEqual<int>(0, process.ExitCode);
+    }
+
+    //[TestMethod]
+    //public void Run_GoogleDotCom_Success()
+    //{
+    //    int exitCode = Sut.Run("-c 8 google.com").ExitCode;
+
+    //    Assert.AreEqual<int>(0, exitCode);
+    //}

```

Comment:
Per teams - I believe this is how this test can be fixed to work w/ github actions
```suggestion
      [TestMethod]
   public void Run_GoogleDotCom_Success()
   {
      int expectedCode = Environment.GetEnvironmentVariable("GITHUB_ACTIONS") is null ? 0 : 1;
      int exitCode = Sut.Run("google.com").ExitCode;
       Assert.AreEqual<int>(expectedCode, exitCode);
   }

```

Reply:
Thanks you, went ahead and implemented this change